services/web/server/src/simcore_service_webserver/projects/utils.py

```diff
@@ -67,8 +67,11 @@ def _replace_uuids(node: str | list | dict) -> str | list | dict:
         if isinstance(node, str):
             # NOTE: for datasets we get something like project_uuid/node_uuid/file_id
             if "/" in node:
-                parts = node.split("/")
-                node = "/".join(_replace_uuids(part) for part in parts)
```

Comment:
What about? Does this not work as expected?
```suggestion
                node = "/".join(f"{_replace_uuids(part)}" for part in parts)
```

Reply:
you are right, i can rewrite it that way thanks